.github/workflows/dependency-updates-post-processing.yaml

```diff
@@ -1,23 +1,8 @@
 name: "Dependency Updates Post-Processing"
 on:
   pull_request:
-    branches:
```

Comment:
Just out of curiosity: How have you tested that? 🤔 💭

Reply:
I ran locally with a cherry-picked depdendabot update commit and `act`.

Reply:
I wasn't aware of something like `act` and that it is possible to test the actions locally.

So far, the feedback loop was quiet long for me, because I always configured "manual workflow triggers" or changed branches etc. to execute GH actions for a particular PR 🙇‍♂️